docker-compose-elk.yml

```diff
@@ -6,7 +6,7 @@ services:
       resources:
         limits:
           memory: 8g
-    image: docker.elastic.co/elasticsearch/elasticsearch:5.4.0
+    image: docker.elastic.co/elasticsearch/elasticsearch:${ELK_VERSION}
```

Comment:
Should we document somewhere that `${ELK_VERSION}` is defined in `.env`? Mayb at the beginning of this yml file?